### PR TITLE
Fixed missing topics

### PR DESCRIPTION
**Pull request by 6l20**
## PR Description

Topics env vars are missing for qa and staging envs

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: environments/common.yaml.gotmpl [hunk 1]
```diff
@@ -39,7 +38,6 @@ orchestrate:
       KAFKA_SASL_MECHANISM: "PLAIN"
       DB_USER: {{ requiredEnv "TANUKI_PG_USER" }}
       DB_HOST: {{ requiredEnv "TANUKI_PG_HOST" }}
-      DB_DATABASE: {{ env "TANUKI_PG_DB" | default "postgres" }}
```
Comment (ggarri) [inline environments/common.yaml.gotmpl:42]: Why did you remove this?

**Review by darioAnongba — APPROVED** [review]
